training/tf/parse.py

```diff
@@ -108,17 +105,14 @@ def _parse_function(planes, probs, winner):
 
 def main(args):
     train_data_prefix = args.pop(0)
+    test_data_prefix = args.pop(0)
 
-    chunks = get_chunks(train_data_prefix)
-    print("Found {0} chunks".format(len(chunks)))
-
-    if not chunks:
+    training = get_chunks(train_data_prefix)
+    test = get_chunks(test_data_prefix)
```

Comment:
This breaks the script from working with v1 input?

Reply:
Yes. I added argument parsing to allow specifying just train files.